src/routes/workshop-a11y-fundamentals.svelte

```diff
@@ -203,174 +550,348 @@
+            <li>{topic}</li>
+          {/each}
+        </ul>
+      </li>
+    {/each}
+  </ul>
+
+
+  <h3 class="t-title asH3">
+    By the end, you will:
+  </h3>
+  <ul class="t-list asCheck">
+    <li>Realize how web accessibility benefits everyone</li>
+    <li>Understand WCAG principles and how is organized</li>
+    <li>Know how to use accessibility audit tools</li>
+    <li>Quickly identify common accessibility issues</li>
+    <li>Integrate accessibility into your team workflow right away</li>
     <li>
       Find out that building inclusive websites isn’t as hard as it sounds
       <span aria-hidden="true">;)</span>
     </li>
   </ul>
 
-  <h2 class="f-title subtitle">Topics</h2>
-  <ul class="list">
-    <li>Demystify Accessibility myths;</li>
-    <li>Disability types spectrum;</li>
-    <li>WCAG - Layers of Guidance;</li>
-    <li>Comformance Levels;</li>
-    <li>Common accessibility no-nos;</li>
-    <li>Semantic markup;</li>
-    <li>Keyboard navigation;</li>
-    <li>Screen readers;</li>
-    <li>ARIA attributes;</li>
-    <li>Landmarks;</li>
-    <li>Accessibility debugging tools;</li>
-    <li>Whatever question you have!</li>
-  </ul>
 
-  <h2 class="f-title subtitle">Modules</h2>
-  <p class="p">The workshop will be given through 3 sessions of 3 hours each.</p>
-  <ol class="list">
-    <li>
-      <strong class="b">
-        <span aria-hidden="true">#1:</span>
-        Accessible by default
-      </strong>
-      <p class="p">Appearance (color, size, motion) and semantics.</p>
-    </li>
-    <li>
-      <strong class="b">
-        <span aria-hidden="true">#2:</span>
-        Keyboard navigation
-      </strong>
-      <p class="p">Interactive elements, hiding techniques, navigation shortcuts.</p>
-    </li>
-    <li>
-      <strong class="b">
-        <span aria-hidden="true">#3:</span>
-        Screen readers
-      </strong>
-      <p class="p">Meaningful content, landmarks, forms and common design patterns.</p>
-    </li>
-  </ol>
-
-  <h2 class="f-title subtitle">Workshop Dynamics</h2>
-  <p class="p">The workshop consists in a set of challenges with detailed resources.</p>
-  <p class="p">
-    Each challenge introduces a new topic, followed by a hands-on exercise for you to apply the
-    concepts learned.
+  <h3 class="t-title asH3">
+    Workshop dynamics
+  </h3>
+  <p class="t-p">
+    In each challenge, I’ll introduce you to a new topic in a simplified way, along with <strong>detailed resources</strong>. 
+  </p>
+  <p class="t-p">
+    Then, you'll have a <strong>hands-on exercise</strong> for you to apply the concepts learned. You can solve it by yourself or by collaborating with a group of 2-3 people.
   </p>
-  <p class="p">
-    After each exercise, we'll go through the solution together and I will clarify any question that
-    you may have.
+  <p class="t-p">
+    Afterwards, we go through the <strong>solution</strong> together, and I’ll <strong>clarify the questions</strong> that you might have.
```

Comment:
Afterwards, we go through the <strong>solution</strong> together, and I’ll <strong>clarify any questions</strong> that you might have.